pygeo/DVGeometry.py

```diff
@@ -2944,6 +3175,87 @@ def _attachedPtJacobian(self, config):
 
         return Jacobian
 
+
+    def _spanwiselocalDVJacobian(self, config=None):
+        """
+        Return the derivative of the coefficients wrt the local normal design
+        variables
+        """
+        # This is relatively straight forward, since the matrix is
+        # entirely one's or zeros
+        nDV = self._getNDVSpanwiseLocalSelf()
+        self._getDVOffsets()
+
+        if nDV != 0:
+            Jacobian = sparse.lil_matrix((self.nPtAttachFull*3, self.nDV_T))
+
+            # Create the storage arrays for the information that must be
+            # passed to the children
+
+            for iChild in range(len(self.children)):
+                N = self.FFD.embededVolumes['child%d_axis'%(iChild)].N
+                self.children[iChild].dXrefdXdvl = numpy.zeros((N*3, self.nDV_T))
+
+                N = self.FFD.embededVolumes['child%d_coef'%(iChild)].N
+                self.children[iChild].dCcdXdvl = numpy.zeros((N*3, self.nDV_T))
+
+            iDVSpanwiseLocal = self.nDVSW_count
```

Comment:
I see most of this comes from `_sectionlocalDVJacobian` but it would be great to have some more inline comments, I am struggling a bit to follow the flow.

Reply:
see new added comments